2 - RTB/Documentazione esterna/Verbali/23-12-15/23-12-15.typ

```diff
@@ -0,0 +1,59 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "15/12/23",
+  subTitle: "",
+  docType: "verbale",
+  authors: (
+    "Nardo Silvio",
+  ),
+  externalParticipants : (
+    (name: "Alberton Michela", role: "Membro azienda proponente"),
+    (name: "Bassani Matteo", role: "Referente aziendale"),
+    (name: "Ruffin Filippo", role: "Membro azienda proponente"),
+  ), 
+  timeStart: "12:00",
+  timeEnd: "12:30",
+);
+
+= Ordine del giorno
+Ai referenti aziendali sono stati esposti i progressi sui PoC rispetto al precedente meeting esterno e sono sorte discussioni relativamente a:
+- Analisi dei Requisiti;
+- tecnologie da utilizzare e utilizzate;
+- feature dei PoC;
+- meeting futuri.
+
+== Considerazioni sull'Analisi dei Requisiti
+Non sono stati identificati particolari vincoli sulle performance.
+
+=== Vincoli sui browser 
+È richiesta la compatibilità dell'applicazione almeno con i browser:
```

Comment:
Scritta così sembra una richiesta esplicita del Proponente, in realtà bisogna far capire che questi requisiti li abbiamo identificati noi sulla base dei requisiti minimi hardware e software richiesti per l'esecuzione di WebGL

Reply:
io ho riportato la richiesta del proponente in seguito alle considerazioni fatte assieme

Reply:
Allora non è chiaro

Reply:
In realtà, trattandosi di requisiti non funzionali, non emergono direttamente dal confronto con proponente, ma bensì sono frutto di analisi nostre. Come dicono Tullio e Cardin, tutti siamo capaci di individuare i requisiti funzionali (che emergono anche dal capitolato o con il Proponente stesso), ma tutta la parte di non funzionali è studio nostro come gruppo. Pertanto, l'individuazione delle versioni dei software non è corretto dire che è stato il proponente ad individuarle, ma bensì ha semplicemente confermato quanto avevamo individuato nella nostra analisi dei requisiti.